neqo-transport/src/packet.rs

```diff
@@ -321,6 +328,8 @@ pub struct PublicPacket<'a> {
     token: &'a [u8],
     /// The size of the header, not including the packet number.
     header_len: usize,
+    /// Protocol version, if present in header.
+    quic_version: Option<Version>,
```

Comment:
Why not DraftVersion?

Reply:
sure, if it's ok that when `decode()` returns `PacketType::OtherVersion` that version is `None`